src/screens/Connectors/ConnectorUtils.res

```diff
@@ -968,16 +995,16 @@ let mapAuthType = (authType: string) => {
   }
 }
 
-let getConnectorType = (connector: ConnectorTypes.connectorTypes, ~isPayoutFlow) => {
-  isPayoutFlow
-    ? "payout_processor"
-    : switch connector {
-      | ThreeDsAuthenticator(_) => "authentication_processor"
-      | PMAuthenticationProcessor(_) => "payment_method_auth"
-      | TaxProcessor(_) => "tax_processor"
-      | UnknownConnector(str) => str
-      | _ => "payment_processor"
-      }
+let getConnectorType = (connector: ConnectorTypes.connectorTypes) => {
+  switch connector {
+  | Processors(_) => "payment_processor"
+  | PayoutConnector(_) => "payout_processor"
+  | ThreeDsAuthenticator(_) => "authentication_processor"
+  | PMAuthenticationProcessor(_) => "payment_method_auth"
+  | TaxProcessor(_) => "tax_processor"
+  | UnknownConnector(str) => str
+  | _ => "unknown"
```

Comment:
can we handle the remaining type that is `FRM` instead of the `_`case,
it will be `| FRM(_) => "payment_vas"`